toolbox/commands/verify.py

```diff
@@ -35,16 +36,29 @@ def verify(context: typer.Context):
             rich.print(f"[red]Missing description file for {subdir_path}")
             continue
 
+        yaml_data = yaml.safe_load(config_path.read_text(encoding='utf-8'))
+
         try:
             # print(yaml.sa
             # fe_load(config_path.read_text(encoding='utf-8')))
-            yaml_data = yaml.safe_load(config_path.read_text(encoding='utf-8'))
             validate(yaml_data, schema)
             valid_count += 1
         except ValidationError as error:
             invalid_count += 1
             rich.print(f"[red]Validation error in {config_path}: {error.message}")
 
+        assets = yaml_data.get("assets", [])
+        if assets:
+            if not assets_path.is_dir():
+                invalid_count += 1
```

Comment:
If one task has 2 errors invalid will be modified 2 times.  I think extract inner code of `for subdir_path in find_tasks(tasks_directory):` to separate function and return boolean descriping if it is valid